src/Microsoft.ML.Tokenizers/Model/Model.cs

```diff
@@ -67,6 +67,12 @@ public abstract class Model
         /// <returns></returns>
         public abstract bool IsValidChar(char ch);
 
+        /// <summary>
+        /// Returns if the first character of the token is part of the actual word or not.
+        /// </summary>
+        /// <param name="token"></param>
+        /// <returns></returns>
+        public abstract bool IsFirstTokenInWord(string token);
```

Comment:
>IsFirstTokenInWord [](http://example.com/codeflow?start=29&length=18)

Do we know if this is going to be useful for all tokenizers models? I am asking to know if we move this to the NasBert which know it is using EnglishRoberta tokenizer. I think we need to add to the Model only the functionality we believe will benefit most tokenizers.

Reply:
We could just move it to NasBert I think. Let me do that.